message-queue-lib/src/cmr/message_queue/queue/sqs.clj

```diff
@@ -299,31 +307,6 @@
   [sns-client exchange-name]
   (.createTopic sns-client (normalize-queue-name exchange-name)))
 
-(defn- topic-conditions
```

Comment:
are these deleted because they are unused?

Reply:
Correct, the tool told me that. Also I followed it up with grep for all of CMR.